src/index.js

```diff
@@ -1,23 +1,38 @@
 import './index.css';
-import { TodoListItems } from './todolist.js';
+import * as todoList from './todolist.js';
+import * as addRemove from './add-remove.js';
+import * as storage from './storage.js';
 
-const tasks = [
-  {
-    description: 'Write article',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Go shop',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Complete 2 coding challenges',
-    completed: false,
-    index: 2,
-  },
-];
+const tasks = storage.getTasksFromStorage();
 
-TodoListItems(tasks);
-// document.body.appendChild(component());
\ No newline at end of file
+todoList.TodoListItems(tasks);
+addRemove.TodoListDelete(tasks);
+const addTask = document.querySelector('.text-area');
+addTask.addEventListener('keydown', (e) => {
+  if (e.keyCode === 13) {
+    const task = {
+      description: addTask.value,
+      completed: false,
+      index: tasks.length,
+    };
+    addRemove.TodoListAdd(tasks, task);
+    addTask.value = null;
+  }
+});
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you also add a feature that prevents the submission of `empty inputs` so a user cannot submit an empty task. I know you can do that :muscle: :muscle: